content/tutorials/_index.md

```diff
@@ -3,7 +3,40 @@ title: Tutorials
 weight: 1
 ---
 
-This part of the docs contains some practical guides for getting starting with both Python and our kit.
+This part of the docs contains some practical guides for getting starting with both Python and the simulator.
 
 * [Basic Movement](basic-movement) - explains how to use the kit to operate a motor
 * [Python: A Whirlwind Tour](python-whirlwind-tour) - a guide to using Python, aimed towards those who haven't used it before
+* [A basic guide to using the simulator](simulator) - a guide to using the we-bot simulator
+
+## Installation
+
+### Prerequisites
+
+You need to download and install Webots R2021b (the download is around 0.5 GB). This is the platform we run our simulation in.
```

Comment:
They don't actually need webots downloaded to use the simulator, just vscode. Perhaps linking out to that and saying they'll get a link to join liveshare at the event?